setup.cfg

```diff
@@ -42,7 +43,7 @@ classifiers =
 include_package_data = True
 zip_safe = False
 packages = sphinx_rtd_theme
-python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
+python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*
```

Comment:
1.0.0 specifies to support 3.4 and up.

By adding this, we should state in the changelog that these versions are dropped. But otherwise, I can't see a big problem. Adding this should **not** break old Python 3.4 and 3.5 versions as they simply won't get sphinx_rtd_theme 1.2 from PyPi because of the constraint here.

Reply:
Oh? https://sphinx-rtd-theme.readthedocs.io/en/1.0.0/development.html#supported-dependencies say "Python | 2.7 or 3.6 or greater"

I'll add a note in the changelog, sure

Reply:
Hmm, Roadmap [says Python >=3.6](https://sphinx-rtd-theme.readthedocs.io/en/stable/development.html#supported-dependencies) and the [Changelog for 1.0.0 says >=3.4](https://sphinx-rtd-theme.readthedocs.io/en/stable/changelog.html#release-1-0-0)

This seems messy - we should state the supported Python version in a way that's perhaps a bit easier to find. I think it's especially for new versions of Python that it's nice to be able to quickly look this up.

Reply:
After this change, the 1.2 changelog entry aligns with the Development/Roadmap dependency specification, so we also got rid of the contradiction :+1: